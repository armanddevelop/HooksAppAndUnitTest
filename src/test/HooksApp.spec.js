import { HooksApp } from "../HooksApp";
import { shallow } from "enzyme";
import "@testing-library/jest-dom";
import toJson from "enzyme-to-json";

describe("Snapshot to HooksApp", () => {
  it("should take a snapShot", () => {
    const wrapper = shallow(<HooksApp />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
