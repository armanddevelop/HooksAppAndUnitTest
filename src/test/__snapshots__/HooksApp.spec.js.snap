// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot to HooksApp should take a snapShot 1`] = `
<div>
  HooksApp
</div>
`;
